cmd/cockroach-operator/BUILD.bazel

```diff
@@ -40,13 +40,13 @@ container_image(
     labels = {
         "name": "CockroachDB Operator",
         "vendor": "Cockroach Labs",
-        # FIXME - labels do not support a stamped variable such as {STABLE_DOCKER_TAG}
-        "version": "v1.0.0-rc.0",
-        "release": "0",
-        "summary": "CockroachDB is a Distributed SQL database",
-        "description": "CockroachDB is a PostgreSQL wire-compatible DistributedSQL database",
+        "version": "{STABLE_DOCKER_TAG}",
+        "release": "{STABLE_DOCKER_TAG}",
```

Comment:
I would prefer to not have a 0 here since it needs to be an up to date release number.  What version number do you think we should use?  The PR number?

Reply:
How about we use the number of commits since the last tag in the git repo?

```
git describe --tags  --match=[TAG_NAME_REGEX] 2> /dev/null | sed -e 's/^.*-\([0-9]*\)-g[0-9a-f]*$/\1/'
```

An example tag name regex (which we use in cockroach): `v[0-9]*`

Reply:
I'm curious why not use the number of commits since the last tag on the current branch? Assuming the tag is already made and the tag is for the current commit, it would usually result in `0` (since there are 0 commits since the current tag). And then, if it's not tagged, it would show the number of commits since whatever the last tag was.

What about one of these:

- Number of commits since the major release was tagged:
    ```
    git describe --tags  --match=[TAG_NAME] 2> /dev/null | sed -e 's/^.*-\([0-9]*\)-g[0-9a-f]*$/\1/'
    ```
- Total number of commits on the current branch: 
    ```
    git rev-list $(git branch --show-current) | wc -l
    ```

The numbers for all commits on master or all commits on all branches are not stable (they'll change with every commit merged to master / every commit added to the repo).

Reply:
`git rev-list $(git branch --show-current) | wc -l`

Reply:
show-current is not working on debian

Reply:
Ah. It only works with git 2.22 and up ([so post](https://stackoverflow.com/questions/6245570/how-to-get-the-current-branch-name-in-git)). This will work for older releases: `git rev-parse --abbrev-ref HEAD`

Reply:
It is another rev-list command that is supported in the older versions of git.  I figured it out.  PTAL